src/coefficients/StdFunctionCoefficient.hpp

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) 2019-2020, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+// # Author: Jonathan Wong @ LLNL.
+
+/**
+    @file StdFunctionCoefficient.hpp
+ */
+
+#ifndef STD_FUNCTION_COEFFICIENT_HPP
+#define STD_FUNCTION_COEFFICIENT_HPP
+#include <functional>
+#include "mfem.hpp"
+
+using namespace mfem;
```

Comment:
Please do not do any using outside of a function if at all.  It confuses where types are coming from.